### PR TITLE
refactoring of manage blog

### PR DESCRIPTION
**Pull request by rizwans-cuelogic**
added confirm message for delete.
updated search for searching tag.
changed add (blog) button to create.

### CODE REVIEW

File: project4/project/ebs/static/ebs/js/edit_blog.js [hunk 1]
```diff
@@ -14,67 +14,80 @@ $(document).ready(function(){
...
+					if($('#file3').text()=="" && $('#file1').text()=="" && $('#file2').text()==""){
+						$('#upload-row').removeClass('form-group');
+					}
+					swal("Deleted!", "Your file has been deleted.", "success");
+	    		}
+	    	});
+		});
+
+
+		// if(confirm('Are you sure to delete this attachment?')){
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/static/ebs/js/edit_blog.js]: Remove unnecessary code

File: project4/project/ebs/static/ebs/js/manage_blog.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ $(document).ready(function() {
            { position:"top center" }
         );
     }
-    $(".search").keyup(function () {
+    /*$(".search").keyup(function () {
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/static/ebs/js/manage_blog.js]: Remove unnecessary code

File: project4/project/ebs/static/ebs/sweetalert/example/example.css [hunk 3]
```diff
@@ -0,0 +1,442 @@
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,300);
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/static/ebs/sweetalert/example/example.css]: Why you added this example css file

File: project4/project/ebs/static/ebs/sweetalert/example/example.scss [hunk 4]
```diff
@@ -0,0 +1,580 @@
+@import url(http://fonts.googleapis.com/css?family=Open+Sans:400,600,700,300); // Open Sans font
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/static/ebs/sweetalert/example/example.scss]: why you need this example.scss file

**Comment by tusharm-cuelogic** [comment]
Remove unnecessary sweetalert js or css file, if not required.

File: project4/project/ebs/static/ebs/js/edit_blog.js [hunk 5]
```diff
@@ -60,32 +60,7 @@ $(document).ready(function(){
 					swal("Deleted!", "Your file has been deleted.", "success");
 	    		}
 	    	});
-		});
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/static/ebs/js/edit_blog.js]: Remove unwanted code

File: project4/project/ebs/static/ebs/js/manage_blog.js [hunk 6]
```diff
@@ -6,32 +6,6 @@ $(document).ready(function() {
            { position:"top center" }
         );
     }
-    /*$(".search").keyup(function () {
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/static/ebs/js/manage_blog.js]: Remove unused code.

File: project4/project/ebs/templates/ebs/manage_blog.html [hunk 7]
```diff
@@ -2,6 +2,7 @@
 {% load static %}
 {% block head_extra %}
 <link rel="stylesheet" type="text/css" href="{% static 'ebs/css/manage_blog.css' %}"/>
+<link rel="stylesheet" type="text/css" href="{% static 'ebs/sweetalert/dist/sweetalert.css' %}">
```
Comment (tusharm-cuelogic) [inline project4/project/ebs/templates/ebs/manage_blog.html:5]: why you included sweetalert.css and JS file in this template file.

Comment (rizwans-cuelogic) [reply project4/project/ebs/templates/ebs/manage_blog.html:5]: i need that for delete in manage blog

File: project4/project/ebs/templates/ebs/manage_blog.html [hunk 8]
```diff
@@ -2,14 +2,15 @@
...
 {% endblock %}
 {% block content %}
 {% for message in messages %}
 	<div id="message"></div>
 {% endfor %}
 <div class="container">
 	<div class="table-responsive">
-  		<table class="table results">
+  		<table id="tb" class="table results">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/manage_blog.html]: Use appropriate names for class and id. follow naming conventions.

File: project4/project/ebs/templates/ebs/manage_blog.html [hunk 9]
```diff
@@ -20,27 +21,26 @@
...
+							<div class="search-wrapper">
+								<input type="text" name="q"
+								class="search form-control" placeholder="Search" value='{{request.GET.q}}'>
+							<select id="filter" class="filter-box form-control" name="filterbox">
+								<option value="al">All</option>
+								<option value="en">Enabled</option>
+								<option value="ds">Disabled</option>
+							</select>
+							<button id="search-bt" class="btn btn-default" type="submit">
+							<i class="glyphicon glyphicon-search"></i></button>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/manage_blog.html]: Remove extra spaces, follow tab indent and remove unwanted enter between tags.

File: project4/project/ebs/templates/ebs/manage_blog.html [hunk 10]
```diff
@@ -20,27 +21,26 @@
...
+							</div>
+						</form>	
 					</th>
 					<th width="90">
-						<a class="btn btn-ternary" href="{% url 'create_blog' %}"><i class="fa fa-plus fa-fw"></i>  Add</a>
+						<a class="btn btn-ternary" href="{% url 'create_blog' %}"><i class="fa fa-plus fa-fw"></i>Create</a>
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/manage_blog.html]: Add space between create and plus icon

File: project4/project/ebs/templates/ebs/manage_blog.html [hunk 11]
```diff
@@ -20,27 +21,26 @@
...
-						<div class="input-group search-wrapper">
-						<input type="text" class="search form-control" placeholder="Search">
-						<div class="input-group-btn">
-							<button class="btn btn-default" type="submit">
-								<i class="glyphicon glyphicon-search"></i>
-							</button>
-						</div>
-						</div>
+						<form  class="form-inline" method="GET" action="">
+							<div class="search-wrapper">
```
Comment (sagarp-cuelogic) [inline project4/project/ebs/templates/ebs/manage_blog.html]: You can use text-center class to align the div in center.

**Comment by sagarp-cuelogic** [comment]
- align all the edit buttons and create button to the right.
 - remove extra spaces from code.

**Comment by sagarp-cuelogic** [comment]
Review comments resolved.

**Pull request merged**